packages/framework-provider-kubernetes/src/services/read-model-registry.ts

```diff
@@ -23,27 +23,44 @@ export class ReadModelRegistry {
     filters: Filters
   ): Promise<Array<ReadModelInterface>> {
     const l = (msg: string): void => logger.debug('readModelRegistry#search: ' + msg)
-    const readModelId = filters.id.eq
-    l(`Got id ${readModelId ?? 'UNDEFINED'}`)
-    if (!readModelId) throw new Error('Only searching by ID is supported')
-    const url = `${this.url}/v1.0/state/statestore/${this.readModelKey(readModelName, readModelId)}`
-    l(`Performing a fetch to ${url}`)
-    const response = await fetch(url)
-    if (!response.ok) {
-      l(`Error on performing GET from ${url}`)
-    }
-    l(`Got result ${JSON.stringify(response)}`)
-    try {
-      const body = await response.json()
-      l(`Got JSON ${JSON.stringify(body)}`)
-      // TODO: Remove unnecessary id setting from k8s read model registry
-      if (body) {
-        body.id = readModelId
+    if (filters.id) {
```

Comment:
We need to discuss this because I think that right now we could remove the ID filter part from this method because if I'm not wrong the filter by ID (the singular query method in graphQL) is managed by the fetch method.

In this case, we are enabling the search by ID in the multiple-element search (plural queries in graphQL), and having id search in this query is quite contradictory because we have a multiple-element query that only allows filter by a unique ID, so we are going to return just one element 😛